xml/System.Net/FileWebRequest.xml

```diff
@@ -717,9 +717,29 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a <see cref="T:System.IO.Stream" /> object for writing data to the file system resource as an asynchronous operation.</summary>
+        <returns>The Task object representing the asynchronous operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+          ## Remarks  
+          > [!NOTE]
+          >  In the case of asynchronous requests, it is the responsibility of the client application to implement its own time-out mechanism.
```

Comment:
Should this be a link to something describing how to implement time-outs?

Reply:
there is an example for [`HttpWebRequest.BeginGetResponse`](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.begingetresponse?view=netcore-3.0) I could adapt it to `FileWebRequest.GetResponseAsync` in case is really needed.

Reply:
Created issue #2672 to address this later.